Protocol of TC meeting at 2024-06-25 (#83)
OpenRailAssociation/technical-committee: meetings/2024-06-25.md

# TC Meeting 2024-06-25

Present: Peter, Stephanie, Loic (from 13:30), Florian, Cornelius; Excused: Max

## Agenda

* Criteria for stage 2 of the incubation process (#78)
  * REUSE (see also https://github.com/OpenRailAssociation/technical-committee/pull/80#issuecomment-2178500733)
  * DCO (see also https://github.com/OpenRailAssociation/technical-committee/pull/80#issuecomment-2178500733)
* Onboarding of Netzgrafik-Editor
* Holidays

## Protocol

* We will switch from preparing the agenda in an issue to prepare it in a pull request with the file we use to describe agenda and protocol
* Requiring REUSE for incubation stage 2
  * https://reuse.software/

>   * Gives license clarity, this is quite important for us as OpenRail Association, as dealing well with open source licensing is one of our selling points

Reviewer: ```suggestion
  * Gives license clarity, this is quite important for us as OpenRail Association, as dealing well with open-source licensing is one of our selling points
```
see https://www.merriam-webster.com/dictionary/open-source

  * Maintainers will check with their teams and we will discuss and decide next week

>   * If we do't adopt REUSE as a criteria we will have to define in another way how to achieve clarity about licensing

Reviewer: ```suggestion
  * If we do not adopt REUSE as a criteria we will have to define in another way how to achieve clarity about licensing
```

* DCO
  * https://developercertificate.org/
  * Example project how to do it: https://www.chef.io/blog/introducing-developer-certificate-of-origin

>   * Introduced by Linux Kernel to prevent against risks from legal attacks against open source code

Reviewer: ```suggestion
  * Introduced by Linux Kernel to prevent against risks from legal attacks against open-source code
```

  * Risk of problems is increasing with more diverse sets of contributors, but also for small projects and projects of one company

>   * Having the DCO raises awareness and signals that we are serious about high quality, also of copyright information

Reviewer: ```suggestion
  * Having the DCO raises awareness and signals that we are serious about high-quality, also of copyright information
```
https://www.merriam-webster.com/sentences/high%20quality

>   * Prominent open source projects which use it: GitLab, Docker, Chef, lot of CNCF projects

Reviewer: ```suggestion
  * Prominent open-source projects which use it: GitLab, Docker, Chef, lot of CNCF projects
```

>   * It adds friction for new contributors who are not used to the DCO. Having a small barrier might also be a good thing to signal that we want high quality contributions

Reviewer: ```suggestion
  * It adds friction for new contributors who are not used to the DCO. Having a small barrier might also be a good thing to signal that we want high-quality contributions
```
